host/xtest/regression_4000.c

```diff
@@ -4150,12 +4164,20 @@ static void xtest_tee_test_4006(ADBG_Case_t *c)
 				num_algo_params = 1;
 			}
 
-
 			out_enc_size = sizeof(out_enc);
-			if (!ADBG_EXPECT_TEEC_SUCCESS(c,
-				ta_crypt_cmd_asymmetric_encrypt(c, &session, op,
+			res = ta_crypt_cmd_asymmetric_encrypt(c, &session, op,
 					algo_params, num_algo_params, tv->ptx,
-					tv->ptx_len, out_enc, &out_enc_size)))
+					tv->ptx_len, out_enc, &out_enc_size);
+
+			if (tv->algo == TEE_ALG_RSAES_PKCS1_OAEP_MGF1_SHA256 &&
+			    res == TEEC_ERROR_NOT_SUPPORTED) {
+				Do_ADBG_Log("RSA-OAEP with a different MGF1 "
+					    "hash not supported: skip "
+					    "subcase");
```

Comment:
Is it worth to split this trace message?
Ditto at line 4258.

Reply:
Probably not, I'm fine with not splitting this.